tests/20-rdfc-verify.js

```diff
@@ -85,7 +68,11 @@ describe('ecdsa-rdfc-2019 (verify)', function() {
             this.test.cell = {
               columnId: `${name}: ${keyTypes}`, rowId: this.test.title
             };
-            for(const credential of credentials) {
+            for(const vector of supportedVectors) {
+              const credential = klona(vector);
+              //FIXME add invalid-cryptosuite as a valid cryptosuite name
+              //locally so the signature is correct, but the cryptosuite
+              //name is incorrect
               credential.proof.cryptosuite = 'invalid-cryptosuite';
```

Comment:
Unrelated to this PR: I dont love how this works

Reply:
@tminard feel free to tackle anything like this you see. PRs welcome and all that. 😁